src/app.js

```diff
@@ -1 +1,21 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const fileCopy = (src, dest) => {
+  if (src === dest || (!src || !dest)) {
+    return;
+  }
+
+  const srcPath = path.join(__dirname, `/${src}`);
+  const destPath = path.join(__dirname, `/${dest}`);
+
+  const data = fs.readFileSync(srcPath, 'utf-8');
```

Comment:
* The readFileSync and writeFileSync methods are blocking the I/O which could potentially lock up the Node.js event loop. Consider using the async versions of these methods for better performance.

* You are reading the entire file into memory before copying. It's blocking and redundant for large files. Consider using streams